apps/articles/client/team_channel.coffee

```diff
@@ -8,13 +8,18 @@ sd = require('sharify').data
 
 module.exports.TeamChannelView = class TeamChannelView extends Backbone.View
 
+  events: ->
+    'click .js-team-channel-toggle-hamburger' : 'toggleHamburgerNav'
+
   initialize: ->
+    @$body = $('body')
     @channel = new Channel sd.CHANNEL
     @gridArticles = new Articles
     @gridArticles.url = "#{@gridArticles.url}/?&published=true&limit=12&sort=-published_at&channel_id=#{@channel.get('id')}"
     @renderGrid()
     @renderFeatured()
     @setupStickyNav()
+    $(window).resize @windowResized
```

Comment:
You _might_ want to throttle this, though the event probably won't happen that often. Developers are the only ones who resize screens :)